### PR TITLE
ZBUG-399: Code fixed to ensure default crontab must not enabled if NG…

### PR DESCRIPTION
**Pull request by srivastava-ami**
… Backup is already initialized with zxsuit check

@Prashantsurana @fatkudu  please check and approve. I added the delay of 20 sec while checking for the zxsuit command because it take bit time after the mailbox restart.

### CODE REVIEW

**Comment by srivastava-ami** [comment]
@Prashantsurana added the subroutine to check if the NG service is up and running.

File: rpmconf/Install/zmsetup.pl [hunk 1]
```diff
@@ -7295,11 +7295,31 @@ sub setupSyslog {
   configLog("setupSyslog");
 }
 
+sub zxsuitIsAvailable {
```
Comment (fatkudu) [inline rpmconf/Install/zmsetup.pl]: typo - s/zxsuit/zxsuite/ ?

File: rpmconf/Install/zmsetup.pl [hunk 2]
```diff
@@ -7295,11 +7295,31 @@ sub setupSyslog {
...
+  progress("done. \n");
+}
+
+
 sub setupCrontab {
   my @backupSchedule=();
   my $nohsm=1;
-  my $NGbackup = qx(/opt/zimbra/bin/zxsuite backup getBackupInfo | awk \'/valid/ {print \$2}\');
-  detail(" we will modify the crontab if NGBackup=false: $NGbackup");
+  my $NGbackup = "false";
```
Comment (Prashantsurana) [inline rpmconf/Install/zmsetup.pl]: Keep everything in zxsuitIsAvailable() 
It should return true or false.

File: rpmconf/Install/zmsetup.pl [hunk 3]
```diff
@@ -7295,11 +7295,31 @@ sub setupSyslog {
...
+  if (-f "/opt/zimbra/bin/zxsuite") {
+  zxsuitIsAvailable();
```
Comment (fatkudu) [inline rpmconf/Install/zmsetup.pl]: Please indent.

**Pull request closed without merging**